src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.ts

```diff
@@ -481,10 +481,7 @@ export class UbsAdminTariffsLocationDashboardComponent implements OnInit, AfterV
       .subscribe((card) => {
         card.forEach((el) => {
           const cardObj = {
-            courier: el.courierTranslationDtos
-              .filter((ob) => ob.languageCode === 'ua')
-              .map((it) => it.name)
-              .join(),
+            courier: el.courierTranslationDtos.map((it) => it.name).join(),
```

Comment:
Will there be translation for eng available?

Reply:
We receive all data for tariff pricing page and its pop-up windows  from the database in Ukrainian language. It is assumed that all admins will be Ukrainian-speaking. Therefore, today there is no need to translate data about the location, courier and receiving station into English. But it probably will next feature for implementing.

Reply:
Well, i feel like it is good to add here if back-end passes both ua and en versions, not sure about it though